src/containers/NFT/NFTHeader/styles.scss

```diff
@@ -0,0 +1,201 @@
+@import '../../shared/css/variables';
+@import '../../shared/css/table';
+
+.section {
+  margin: 0 32px;
+  @include for-size(tablet-landscape-up) {
+    width: 552px;
+    margin: 0px auto;
+  }
+
+  @include for-size(desktop-up) {
+    width: 1140px;
+  }
+}
+
+.nft-header-container {
+  .nft-bottom-container {
+    display: flex;
+    flex-direction: column;
+
+    @include for-size(desktop-up) {
+      flex-direction: row;
+    }
+
+    .details {
+      width: 100%;
+      @include for-size(desktop-up) {
+        width: 490px;
+      }
+    }
+
+    .settings {
+      width: 100%;
+      @include for-size(desktop-up) {
+        width: 650px;
+      }
+    }
+
+    .title {
+      padding: 64px 0 18px;
+      border-bottom: 1px $gray-800 solid;
+      color: $white;
+      font-size: 24px;
+      @include bold;
+
+      @include for-size(desktop-up) {
+        padding-top: 80px;
+      }
+    }
+  }
+
+  .nft-info-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 68px;
+
+    @include for-size(desktop-up) {
+      flex-direction: row;
+      margin-top: 80px;
+    }
+
+    .values {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      margin-bottom: 18px;
+      color: $white;
+
+      @include for-size(desktop-up) {
+        flex-direction: column;
+        margin-bottom: 0;
+      }
+
+      .title {
+        padding-bottom: 4px;
+        margin-bottom: 4.5px;
+        color: $gray-400;
+        font-size: 14px;
+        text-transform: uppercase;
+        @include semibold;
+      }
+
+      .value {
+        display: flex;
+
+        .value-content {
```

Comment:
this class could be named `content` since the `value` part is inferred from the nesting

Reply:
ahh yes, but ill probably rename it to something else, since the 'content' classname is conflicting with another style